script/elevate-cpanel.PL

```diff
@@ -1052,6 +1054,7 @@ sub run_stage_2 ($self) {
 
     Elevate::Motd->setup();
 
+    $self->run_component_once( 'AutoSSL'    => 'pre_leapp' );
```

Comment:
The 'SSH' component should still get executed before this.  Can we move this down one line please?

Reply:
Okay, I'll move it to run after the SSH component.